tools/devtool

```diff
@@ -969,6 +969,23 @@ check_swap () {
 	say_warn "WARNING: SWAP enabled"
 }
 
+check_SSBD () {
+    arch=$(uname -m)
+    if [ "$arch" = "aarch64" ]; then
+        local param="ssbd=force-on"
+    elif [ "$arch" = "x86_64" ]; then
+        local param="spec_store_bypass_disable=on"
+    fi
+
+    ssbd_sysfs_file="/sys/devices/system/cpu/vulnerabilities/spec_store_bypass"
+
+
+    (test -f $ssbd_sysfs_file) && (grep -q "^Vulnerable" $ssbd_sysfs_file) && \
+      say_warn "WARNING: SSBD mitigation is either globally disabled or"\
+      "system does not support mitigation via prctl or seccomp. Try enabling"\
+      "it system-wide, using the \`${param}\` boot parameter."
```

Comment:
This doesn't warn if `ssbd_sysfs_file` doesn't exist

Reply:
If this file is not present, the kernel does not support this mitigation. I think this shouldn't happen on kernels newer than 4.14. Is this correct @georgepisaltu ?

Reply:
I'll update this nevertheless, to be on the safe side

Reply:
yes, pls cover that case even if it's not supported, it only costs us one line here 😄